src/index.js

```diff
@@ -7,6 +8,7 @@ export { default as Section } from "./components/basic/containers/Section";
 /** Form Components */
 export { default as Checkbox } from "./components/basic/form/Checkbox";
 export { default as CheckedInput } from "./components/basic/form/CheckedInput";
+export {default as  Images} from "./components/site/Images";
```

Comment:
We won't need to export this component, as we we handle this in the prototype as a page.

Reply:
done